cdap-metadata-spi/src/main/java/io/cdap/cdap/spi/metadata/MetadataExporter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.spi.metadata;
+
+import io.cdap.cdap.api.annotation.Beta;
+
+/**
+ * Interface for exporting metadata into different formats.
```

Comment:
What do mean by different format? I think this interface is more a LineageInfoConsumer?

Reply:
Renamed to `MetadataConsumer` because we might want to also export metadata for programs